step-functions/step-functions-docker-handler/src/main/java/step/grid/ProxyGridServices.java

```diff
@@ -24,7 +24,7 @@ public class ProxyGridServices {
     @Path("/file/{id}/{version}")
     public Response getFile(@PathParam("id") String id, @PathParam("version") String version) throws IOException, FileManagerException {
```

Comment:
@david-stephan Don't we want to expose the new cleanable parameter in this service? doesn't seem to be required right now but could be later.

Reply:
@jeromecomte : I added it as optional query parameter with default value true.